evm/vm/forks/byzantium/computation.py

```diff
@@ -0,0 +1,28 @@
+from cytoolz import (
+    merge,
+)
+
+from evm import precompiles
+from evm.utils.address import (
+    force_bytes_to_address,
+)
+from evm.vm.forks.frontier.computation import FRONTIER_PRECOMPILES
+from evm.vm.forks.spurious_dragon.computation import SpuriousDragonComputation
+
+from .opcodes import BYZANTIUM_OPCODES
+
+BYZANTIUM_PRECOMPILES = merge(
+    FRONTIER_PRECOMPILES,
+    {
+        force_bytes_to_address(b'\x05'): precompiles.modexp,
```

Comment:
Note to self.  This function might be better named something like:

- `expand_bytes_to_address` 
- `pad_bytes_to_address`
- `pad_to_20_byte_address` <-- maybe most explicit